hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -560,6 +560,67 @@
     }
   ]
 
+  features: [
+    {
+      name: "OTP_CTRL.VENDOR_TEST_PARTITION"
+      desc: '''Vendor test partition is intended to be used for OTP programming smoke check during manufacturing flow.
+      In this partition, uncorrectable errors will not lead to fatal errors and alerts.
+      '''
+    }
+    {
+      name: "OTP_CTRL.CALIBRATION_TEST"
```

Comment:
Is this really a feature of the OTP_CTRL? Or should this rather be a manufacturing test for the ROM / AST interaction?

Reply:
It is not coming from the OTP_CTRL. Maybe I should replace with the exact partition name (CREATOR_SW_CFG).

Reply:
Yeah that is probably better...